### DISCUSSION TITLE
How to update raw.githubusercontent.com remote base branches (e.g., argocd)

### DISCUSSION THREAD

**Discussion by joebowbeer** [renovatebot/renovate#17318] [Request Help] [answered]
I am using a kustomize remote base like the following to install [ArgoCD](https://github.com/argoproj/argo-cd/releases):

https://raw.githubusercontent.com/argoproj/argo-cd/v2.3.0/manifests/install.yaml

The pattern appears to be:

https://raw.githubusercontent.com/OWNER/REPO/TAG/manifests/install.yaml

but Renovate is not seeing the v2.4.x releases

Are raw content links supposed to work?

If not, can someone point me to the associated RFE issue and/or suggest a workaround?

**Comment by viceice** [comment]
not supported, try the regex manager

**Reply by joebowbeer** [reply]
Thanks for the clarification.

Does anyone have a Regex Manager for `raw.githubusercontent.com` URLs?

**Reply by MaronHatoum** [reply]
Hi @joebowbeer 
try to add `regexManagers` to your configuration.

```
{
  "regexManagers": [
    {
      "fileMatch": ["(^|\/|\\.)(install\.yaml)$"],
      "matchStrings": ["\s*image:\s+?(?<depName>[a-z\/.-]+)(?::(?<currentValue>[a-z0-9.-]+))?"],
    }
  ]
}
```
**More details at** https://docs.renovatebot.com/modules/manager/regex/

**Comment by joebowbeer** [comment, marked as answer]
The following regexManager handles the additional github links in my kustomization files:

```json
{
  "regexManagers": [
    {
      "fileMatch": ["(^|/)kustomization\\.ya?ml$"],
      "matchStringsStrategy": "any",
      "matchStrings": [
        "https://github.com/(?<depName>[\\w-]+/[\\w-]+)/raw/(?<currentValue>[\\w\\.-]+)//",
        "https://raw.githubusercontent.com/(?<depName>[\\w-]+/[\\w-]+)/(?<currentValue>[\\w\\.-]+)/",
        "https://github.com/(?<depName>[\\w-]+/[\\w-]+)/releases/download/(?<currentValue>[\\w\\.-]+)/"
      ],
      "datasourceTemplate": "github-releases"
    }
  ]
}
```
